src/listopia/list/handler.clj

```diff
@@ -37,7 +37,9 @@
 (defn handle-delete-list! [req]
   (let [db      database-url
         list-id (java.util.UUID/fromString (:list-id (:route-params req)))
-        exists? (list.model/delete-list! db {:list-id list-id})]
+        exists? (do
```

Comment:
`do` will run both actions but only return the 2nd which is what we're interested in for the value of exists.